features/supplier/supplier_applies_for_an_opportunity.feature

```diff
@@ -269,3 +269,16 @@ Scenario: Supplier changes their answers before submission
       | Talk snobbishly about water quality | First nice to have evidence |
       | Sip quietly                         |                             |
       | Provide biscuits                    |  Only the finest            |
+
+@opportunity-clarification-question
+Scenario: Supplier asks a clarification question
+  Given that supplier is on that framework
+  And that supplier has a service on the digital-specialists lot
+  And I have a live digital-specialists brief
```

Comment:
`And I have a live digital-specialists brief` is in the Background so we probably don't need to duplicate it here.